tests/unit/modules/test_saltcheck.py

```diff
@@ -321,6 +322,42 @@ def test__assert_less_equal3(self):
             mybool = sc_instance._SaltCheck__assert_less_equal(aaa, bbb)
             self.assertEqual(mybool, 'Pass')
 
+    def test__assert_empty(self):
```

Comment:
It's great that we have additional tests here.

Maybe I'm missing something, but I'm not seeing tests for the new runtests functionality (saltenv, checkall).

Reply:
This is part of a series of updates for the saltcheck module, the latest of which is #53068. Might it make sense to allow these ports into master, and handle the doc and test updates as part of the most recent PR?

Reply:
Maybe?

Should #53068 to point at master?